docs/coordinates/solarsystem.rst

```diff
@@ -129,6 +130,14 @@ For a list of the bodies for which positions can be calculated, do:
     to assume that ``get_body(time, 'sun')`` and ``get_sun(time)`` will give
     the same result.
 
+.. note :: 
+    When using JPL ephemerides, be aware that answers may change at levels that
+    can be surprising if you are not careful about understanding the frame you
+    are in or .  See for example the case of the DE440s ephemerides, which is
```

Comment:
Weird end of sentence. Just delete "or"?

Reply:
Yep good point!  Will do:
```suggestion
    are in.  See for example the case of the DE440s ephemerides, which is
```